### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by jpzwarte**

### CODE REVIEW

File: .changeset/odd-adults-change.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@sl-design-system/icon": major
```
[the remark this answers is not in the document]
Comment (anna-lach) [reply .changeset/odd-adults-change.md:2]: https://github.com/sl-design-system/components/issues/1254

File: .changeset/many-dragons-approve.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+"@sl-design-system/checkbox": major
+"@sl-design-system/form": major
```
[the remark this answers is not in the document]
Comment (jpzwarte) [reply .changeset/many-dragons-approve.md:3]: Form-field is in the same package as form.

File: .changeset/odd-adults-change.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpzwarte) [reply .changeset/odd-adults-change.md:2]: No, i believe we discussed it being stable during the meeting this morning :)

File: packages/components/badge/src/badge.stories.ts [hunk 3]
```diff
@@ -38,7 +38,7 @@ const sizeName = (size: string): string => {
 
 export default {
   title: 'Components/Badge',
-  tags: ['stable'],
+  tags: ['preview'],
```
[the remark this answers is not in the document]
Comment (jpzwarte) [reply packages/components/badge/src/badge.stories.ts:41]: Done

**Review by Diaan — APPROVED** [review]

**Pull request merged**